inc/sitemaps/class-sitemaps-renderer.php

```diff
@@ -39,11 +39,11 @@ class WPSEO_Sitemaps_Renderer {
 	protected $needs_conversion = false;
 
 	/**
-	 * Timezone.
+	 * The date helper.
 	 *
-	 * @var WPSEO_Sitemap_Timezone
+	 * @var WPSEO_Date_Helper
 	 */
-	protected $timezone;
+	protected $date;
```

Comment:
Minor: at first glance I was expecting `$date` to be a date object, instead it's the `WPSEO_Date_Helper` instance. Maybe consider a better naming to improve clarity.

Reply:
I understand. We decided to leave the helper from the naming because that doesn't add anything and it reads better: `$this->date->format(...)`

Reply:
Makes sense, thanks!